### PR TITLE
Prep 2.0.0 release

### PR DESCRIPTION
**Pull request by suhsteve**
This PR prepares the v2.0.0 release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,14 +39,18 @@
     <tbody align="center">
         <tr>
             <td >2.3</td>
-            <td rowspan=3><a href="https://github.com/dotnet/spark/releases/tag/v1.1.1">v1.1.1</a></td>
+            <td rowspan=1><a href="https://github.com/dotnet/spark/releases/tag/v1.1.1">v1.1.1</a></td>
```
[the remark this answers is not in the document]
Comment (imback82) [reply README.md:42]: yes, let's remove this.

File: docs/release-notes/2.0.0/release-2.0.0.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+
+* Add 'Z' to the string format in Timestamp.ToString() to indicate UTC time ([#897](https://github.com/dotnet/spark/pull/897))
+
+### Infrastructure / Documentation / Etc.
+
+* None
+
+### Breaking Changes
+
+* None
```
Comment (imback82) [inline docs/release-notes/2.0.0/release-2.0.0.md:27]: #897 was a breaking change and that's the reason we are bumping to 2.0?

Comment (suhsteve) [reply docs/release-notes/2.0.0/release-2.0.0.md:27]: Should we move it here instead ?

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (suhsteve) [reply README.md]: removed.

File: docs/release-notes/2.0.0/release-2.0.0.md (hunk 2 above)
Comment (suhsteve) [reply docs/release-notes/2.0.0/release-2.0.0.md]: Moved here.

**Review by imback82 — APPROVED** [review]
LGTM, thanks @suhsteve!

**Pull request merged**